scripts/instance-settings.generated.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable quotes, comma-dangle */
```

Comment:
should this be commited in repo if it's generated?

Reply:
Good point, I've removed it.